bitswap.go

```diff
@@ -265,14 +265,14 @@ func (bs *Bitswap) GetBlocks(ctx context.Context, keys []cid.Cid) (<-chan blocks
 // HasBlock announces the existence of a block to this bitswap service. The
 // service will potentially notify its peers.
 func (bs *Bitswap) HasBlock(blk blocks.Block) error {
-	return bs.receiveBlocksFrom("", []blocks.Block{blk})
+	return bs.receiveBlocksFrom(nil, "", []blocks.Block{blk})
```

Comment:
nit: this should just pass `context.Background()`. Nil contexts are heavily discouraged in go.